service-loadbalancer/service_loadbalancer.go

```diff
@@ -436,13 +467,19 @@ func main() {
 	if len(*tcpServices) == 0 {
 		glog.Infof("All tcp/https services will be ignored.")
 	}
-	go healthzServer()
-
-	proc.StartReaper()
 
 	var kubeClient *unversioned.Client
 	var err error
 
+	errorPage, err := newStaticPageHandler(*defStaticPage)
```

Comment:
I just noticed a bug while reviewing, you're free to not fix it and I'll do so with a follow up pr. If you're ok fixing it, please move the flags.Parse from the first line of main() to after we're parsed the clientConfig, ie: 
```
clientConfig := kubectl_util.DefaultClientConfig(flags)
flags.Parse(os.Args)
```

This will allow use of the flag overrides defined by the client config (https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/clientcmd/overrides.go#L105)